root/static/scripts/relationship-editor/generic.js

```diff
@@ -101,8 +101,18 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
         release: function (relationships, series) {
             return _.sortBy(
                 relationships,
-                function (r) { return _(r.target(series).events).map(getDate).sort().head() },
-                function (r) { return _(r.target(series).labels).map(getCatalogNumber).sort().head() }
+                function (r) { 
+                    return _(r.target(series).events)
+                        .map(getDate)
+                        .sort()
+                        .head(); 
+                },
+                function (r) { 
+                    return _(r.target(series).labels)
+                        .map(getCatalogNumber)
+                        .sort()
+                        .head(); 
+                }
```

Comment:
(this is to satisfy our comma-dangle rule)
```suggestion
                },
```